lib/order.rb

```diff
@@ -0,0 +1,18 @@
+require 'menu'
+
+class Order
+  attr_reader :menu
+  
+  def initialize(menu = Menu.new)
+    @menu = menu
+    @order =[]
+  end
```

Comment:
Good idea to use dependency injection to get the menu class and order class interacting. 👍🏻 

This will help later down the line when you want to give your takeaway_spec all the information it would get from the menu class - without it having to ask the menu class (through mocking/doubles).